pkg/kernel/networkservice/ethernetcontext/common.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 Nordix Foundation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ethernetcontext
+
+import (
+	"net"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+
+	"github.com/networkservicemesh/sdk-kernel/pkg/kernel/networkservice/vfconfig"
+)
+
+func setupEthernetConfig(vfConfig *vfconfig.VFConfig, conn *networkservice.Connection, isClient bool) error {
```

Comment:
Probably this file should be `vf_common` instead of just `common`, because it is applicable only to the VF interface.
And so it would be better to name the method `vfCreate` instead of `setupEthernetConfig` to be consistent with the existing naming in `connectioncontextkernel`.

Reply:
yes, done!